driverapi/driverapi.go

```diff
@@ -72,6 +72,11 @@ type Driver interface {
 	// only invoked for the global scope driver.
 	EventNotify(event EventType, nid string, tableName string, key string, value []byte)
 
+	// DecodeTableEntry passes the driver a key, value pair from table it registered
+	// with libnetwork and expects the driver to return a {endpoint ID, info string}
```

Comment:
I think this function is generic enough and could support the decode for any other hypothetical tables that driver might use in future, besides the endpoint_table. Therefore I would not say in the comment that the driver has to return the `endpoint ID` in the tuple.

Actually, could there be a case where the first element of the returned tuple would differ from the input parameter `key` ? Was wondering whether we need the decode function to return again the key or just the decoded value.